pkg/controller/core.oam.dev/v1alpha2/application/application_controller_test.go

```diff
@@ -5196,3 +5198,146 @@ spec:
 
 `
 )
+
+func Test_hasHealthCheckPolicy(t *testing.T) {
```

Comment:
Let's not add this unit test because the health-check policy is deprecated now. Other ones are okay.